src/averageAges.js

```diff
@@ -15,11 +15,19 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  let count = 0;
+  let sumAges = 0;
+
+  for (const { sex, died, born } of people) {
```

Comment:
// learn how to use array methods like .filter .map .some .every .find .reduce
 // avoid using loop and forEach
 // replace `if ()` statement with &&, || or ?: